documentation/tutorials/get-started.md

```diff
@@ -530,55 +546,76 @@ update :assign do
+
+Use `recompile` to pick up changes you've made to your code, or just restart your session.
 
 ```elixir
-# Use this to pick up changes you've made to your code, or restart your session
 recompile()
+```
 
-# Open a ticket
+### Open a Ticket
+
+```elixir
 ticket = (
   Helpdesk.Support.Ticket
   |> Ash.Changeset.for_create(:open, %{subject: "I can't find my hand!"})
   |> Helpdesk.Support.create!()
 )
+```
+
+### Create a Representative
 
-# Create a representative
+```elixir
 representative = (
   Helpdesk.Support.Representative
   |> Ash.Changeset.for_create(:create, %{name: "Joe Armstrong"})
   |> Helpdesk.Support.create!()
 )
+```
 
-# Assign that representative
+### Assign that Representative to the Ticket
+
+```elixir
 ticket = (
   ticket
   |> Ash.Changeset.for_update(:assign, %{representative_id: representative.id})
   |> Helpdesk.Support.update!()
 )
 ```
 
+> Do we use the last assigned ticket?
+> When we say the "The representative itself is not modified in any way" isn't it updating both sides of the relationship?
+
 ### What next?
 
-What you've seen above constitutes some very simple usage of Ash, barely scratching the surface. In a lot of ways, it will look very similar to other tools that you've seen. If all that you ever used was the above, then realistically you won't see much benefit to using Ash over other options. Where Ash shines is in all of the additional turn-key tools that are built in, the ability to extend the framework yourself, and the consistent design patterns that enable unparalleled velocity, power and flexibility as your application and needs grow.
+What you've seen above barely scratches the surface of what Ash can do. In a lot of ways, it will look very similar to other tools that you've seen. If all that you ever used was the above, then realistically you won't see much benefit to using Ash over other options like Ecto.
+
+> Long awkward sentence
+
+Where Ash shines is in all of the tools that can operate on your resources, the ability to extend the framework yourself, and the consistent design patterns that enable unparalleled efficiency, power and flexibility as your application and needs grow.
 
 #### Clean up your code that uses Ash?
 
 Creating and using changesets can be verbose. Check out the {{link:ash:guide:Code Interface}} to derive things like `Helpdesk.Support.Ticket.assign!(representative.id)`
 
+> Not sure what this bit is about, needs expanding
+
 #### Persist your data
 
-See {{link:ash_postgres:guide:Get Started With Postgres:AshPostgres}} to see how to back your resources with postgres. This is highly recommended, as the postgres data layer provides tons of advanced capabilities.
+See {{link:ash_postgres:guide:Get Started With Postgres:AshPostgres}} to see how to back your resources with Postgres. This is highly recommended, as the Postgres data layer provides tons of advanced capabilities.
 
 #### Add an API
 
+> Add links, how should we link to extensions?
```

Comment:
`{{link:ash_postgres:extension:AshPostgres.DataLayer}}` would do it, for example.

Reply:
But the focus has to remain on `ash_hq`, I think. And if people aren't reading the docs there we should improve the experience until they do, IMO.

Reply:
Agreed but the hex experience should be equivalent as that's what Elixir users are used to

Reply:
The ship has sailed on that one, we already use the link expansion all over the place.

Reply:
So it's gotta be a separate project to add a ex doc pre processor

Reply:
Aside from link expansion, though, the experience should be equivalent, so all we need is that one pre processor